pkg/apis/status/v1/types.go

```diff
@@ -0,0 +1,15 @@
+package v1
+
+import (
+	"github.com/submariner-io/submariner/pkg/cable"
```

Comment:
This creates an import loop...

Reply:
That's true, but it's validating and compiling. Did you experience issues while trying to use it externally?